spec.html

```diff
@@ -36185,7 +36213,8 @@ <h1>String.prototype [ %Symbol.iterator% ] ( )</h1>
         <p>This method returns an iterator object that iterates over the code points of a String value, returning each code point as a String value.</p>
         <p>It performs the following steps when called:</p>
         <emu-alg>
-          1. Let _O_ be ? RequireObjectCoercible(*this* value).
+          1. Let _O_ be the *this* value.
+          1. Perform ? RequireObjectCoercible(_O_).
```

Comment:
Ditto.

Reply:
I think this comment is in the wrong place, but I've fixed the other occurrence of the above issue.